mass_mailing_list_dynamic/README.rst

```diff
@@ -0,0 +1,79 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+==========================
+Dynamic Mass Mailing Lists
+==========================
+
+Without this addon you have to choose between providing a dynamic domain and
+letting your mass mailings reach all partners that match it, or being able to
+unsubscribe to certain mailing lists while still being subscribed to others.
+
+This addon allows you to create dynamic mailing lists, so you can now benefit
+from both things.
+
+Usage
+=====
+
+To create a dynamic mailing list, you need to:
+
+#. Go to *Mass Mailing > Mailings > Mailing Lists > Create*.
+#. Choose a *Sync method*.
+#. Choose a *Sync domain*.
```

Comment:
Explain what is a domain. Domain is not a user word. It shouldn't even go with that word in the interface. Look at mass mailings domain field how is called.

Reply:
It says "Domain" there too. The field includes a *help* attribute that explains its meaning, but I'll add the doc here too.